cmd/terramate/cli/cloud.go

```diff
@@ -78,12 +78,43 @@ func (c *cli) checkSyncDeployment() {
 		fatal(err)
 	}
 
-	if orgs := c.cred().organizations(); len(orgs) != 1 {
+	// at this point we know user is onboarded, ie has at least 1 organization.
+	orgs := c.cred().organizations()
+
+	useOrgName := os.Getenv("TM_CLOUD_ORGANIZATION")
+	if useOrgName != "" {
+		var useOrgUUID string
+		for _, org := range orgs {
+			if org.Name == useOrgName {
+				if org.Status != "active" {
+					fatal(errors.E("selected organization %s is not active", useOrgName))
+				}
+
+				useOrgUUID = org.UUID
+				break
+			}
+		}
+
+		if useOrgUUID == "" {
+			fatal(errors.E("organization %q is not in the set of user's organizations: %s",
+				useOrgName,
+				orgs,
+			))
+		}
+
+		c.cloud.run.orgUUID = useOrgUUID
+	} else if len(orgs) != 1 {
 		fatal(
-			errors.E("requires 1 organization associated with the credential but %d found: %s",
-				len(orgs),
-				orgs),
+			errors.E("if the organization is not explicitly set with TM_CLOUD_ORGANIZATION "+
+				"environment variable then the user must be associated with only 1 organization "+
+				"but %d found: %s", len(orgs), orgs),
```

Comment:
```suggestion
			errors.E("Please set TM_CLOUD_ORGANIZATION environment variable to a specific available organization %s", len(orgs), orgs),
```

Reply:
improved in a separate commit because the formatting is wrong